<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Shubham" failures="1" tests="20" name="Default test" time="158.14" errors="0" timestamp="2023-12-14T10:36:32 IST">
  <testcase classname="TestCases.VerifyMultipleCredentialsTest" name="LoginToApplicationWithMultipleDataTest" time="0.472"/>
  <testcase classname="TestCases.VerifyMultipleCredentialsTest" name="LoginToApplicationWithMultipleDataTest" time="0.426">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [https://www.saucedemo.com/] but found [https://www.saucedemo.com/inventory.html]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [https://www.saucedemo.com/] but found [https://www.saucedemo.com/inventory.html]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at TestCases.VerifyMultipleCredentialsTest.LoginToApplicationWithMultipleDataTest(VerifyMultipleCredentialsTest.java:37)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- LoginToApplicationWithMultipleDataTest -->
  <testcase classname="TestCases.VerifyMultipleCredentialsTest" name="LoginToApplicationWithMultipleDataTest" time="0.495"/>
  <testcase classname="TestCases.VerifyMultipleCredentialsTest" name="LoginToApplicationWithMultipleDataTest" time="5.556"/>
  <testcase classname="TestCases.VerifyMultipleCredentialsTest" name="LoginToApplicationWithMultipleDataTest" time="0.5"/>
  <testcase classname="TestCases.VerifyMultipleCredentialsTest" name="LoginToApplicationWithMultipleDataTest" time="0.492"/>
  <testcase classname="TestCases.LoginTestPage" name="loginToApplication" time="1.584"/>
  <testcase classname="TestCases.LoginTestPage" name="verifyTitleofApplication" time="0.095"/>
  <testcase classname="TestCases.LoginTestPage" name="verifyURLofApplication" time="0.069"/>
  <testcase classname="TestCases.CheckOutPage_1_Test" name="inputInformationTest" time="0.432"/>
  <testcase classname="TestCases.CheckOutPage_1_Test" name="verifyTitleOfApplicationTest" time="0.034"/>
  <testcase classname="TestCases.CheckOutPage_1_Test" name="verifyURLCheckOutPage1Test" time="0.012"/>
  <testcase classname="TestCases.Car_Page_Test" name="verifyLableCartPageTest" time="0.099"/>
  <testcase classname="TestCases.Car_Page_Test" name="verifyURLOfTheApplication" time="0.078"/>
  <testcase classname="TestCases.CheckOutPage_2_Test" name="clickOnFinishBtnTest" time="0.081"/>
  <testcase classname="TestCases.CheckOutPage_2_Test" name="verifyLableTest" time="0.032"/>
  <testcase classname="TestCases.Inventory_Page_Test" name="add6productTest" time="3.829"/>
  <testcase classname="TestCases.Inventory_Page_Test" name="remove2ProductTest" time="6.968"/>
  <testcase classname="TestCases.Inventory_Page_Test" name="verifyTwitterLogoTest" time="3.093"/>
  <testcase classname="TestCases.Inventory_Page_Test" name="verifyproductLableTest" time="3.139"/>
</testsuite> <!-- Default test -->
